.github/workflows/release.yml

```diff
@@ -0,0 +1,48 @@
+
+name: Release packages
+
+on:
+  workflow_run:
+    workflows: ["Run tests"]
+    branches: [master]
+    types:
+      - completed
+
+jobs:
+  list_packages:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v2
+      - id: set-matrix
+        run: echo "::set-output name=matrix::$(find thumbor_plugins -type f -name 'setup.py' | awk -F\/ '{print $2"/"$3}' | jq -Rsc '. / "\n" - [""]')"
+  release:
+    name: ${{ matrix.package }}
+    runs-on: ubuntu-latest
+    needs: list_packages
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    strategy:
+      matrix:
+        package: ${{fromJSON(needs.list_packages.outputs.matrix)}}
+    defaults:
+      run:
+        working-directory: ./thumbor_plugins/${{ matrix.package }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: Prepare build environment
+        run: |
+          pip install -U pip
+          pip install -U build
+      - name: Build package
+        run: python -m build
+      - name: Publish a Python distribution to PyPI
```

Comment:
Not sure I like publishing every build @guilhermef... I think releases should have some ceremony. A tag in git, a releases readme  here in github so people can follow what changed in each version. WDYT?

Reply:
If you think publishing every release is useful I would add '-nightly' to the release name, so users know that this is not a stable release, but a release of the main branch.

Reply:
Technically, it will not publish on every build unless the version has changed.
The PyPI upload will ignore if the version is already published.

Reply:
But then how will we know what code corresponds to the version in pypi?

Reply:
I'll add an action to create the tag automatically